app/services/geocode_service.rb

```diff
@@ -2,7 +2,7 @@ class GeocodeService
   attr_reader :connection
 
   def initialize
-    @connection ||= Faraday.new(url: "https://maps.googleapis.com/maps/api/geocode")
+    @connection = Faraday.new(url: "https://maps.googleapis.com/maps/api/geocode")
```

Comment:
Maybe pull the URL out into a variable that we interpolate? It would be a good idea if we need to make different calls